doc/sphinx/docs/python/tutorials/hypervolume.rst

```diff
@@ -148,7 +148,7 @@ Once the hypervolume object is created, it allows for the computation of the fol
 
   >>> # hv and ref_point refer to the data above
   >>> hv.contributions(ref_point) 
-  array([ 0.5 ,  0.25,  0.5 ,  0.  ])
+  array([0.5 , 0.25, 0.5 , 0.  ])
```

Comment:
@darioizzo @CoolRunning are these doctest changes cosmetic only or they are necessary to have the doctests passing?

I am a bit wary of relying on the exact string output of printing an array, as we don't have any control on how numpy decides to print array and that's gonna result in things breaking depending on the numpy version etc.

Reply:
same here. I have added IGNORE_WHITESPACE directive to improve the situation, but indeed it sucks to rely on other paskages to not change their conventions.